JS.md

```diff
@@ -615,13 +599,22 @@ with delta `d`. On failure, a
 [`RangeError`](https://tc39.github.io/ecma262/#sec-native-error-types-used-in-this-standard-rangeerror)
 is thrown.
 
-Perform [`DetachArrayBuffer`](http://tc39.github.io/ecma262/#sec-detacharraybuffer)(`M.[[BufferObject]]`).
-
-Assign to `M.[[BufferObject]]` a new `ArrayBuffer` whose
-[[[ArrayBufferData]]](http://tc39.github.io/ecma262/#sec-properties-of-the-arraybuffer-prototype-object)
-aliases `M.[[Memory]]` and whose 
-[[[ArrayBufferByteLength]]](http://tc39.github.io/ecma262/#sec-properties-of-the-arraybuffer-prototype-object)
-is set to the new byte length of `M.[[Memory]]`.
+If `M.[[BufferObject]]` is a not Shared Data Block:
+  1. Assign to `M.[[BufferObject]]` a new `ArrayBuffer` whose
+     [[[ArrayBufferData]]](http://tc39.github.io/ecma262/#sec-properties-of-the-arraybuffer-prototype-object)
+     aliases `M.[[WebAssembly.Memory]]` and whose
+     [[[ArrayBufferByteLength]]](http://tc39.github.io/ecma262/#sec-properties-of-the-arraybuffer-prototype-object)
+     is set to the new byte length of `M.[[WebAssembly.Memory]]`.
+  2. Do the same in all Shared Data Block which mirror `M.[[BufferObject]]`,
+     doing so atomically in relation to other updates of such blocks.
+Otherwise:
```

Comment:
nit: markdown puts this at the end of item 2, I think you need a \n or something to render correctly